requirements.txt

```diff
@@ -1 +1 @@
-Theano==0.7
+git+https://github.com/Theano/Theano.git@4dacb9e7d04d842d8ffe2c948dbc90722ae990d4#egg=Theano
```

Comment:
0.7 is months old at this point, and this feature doesn't exist in it. This is why I think releases are a bad idea :)